src/ConnectedMode/SlCoreConnectionAdapter.cs

```diff
@@ -63,13 +68,39 @@ public async Task<ValidateConnectionResponse> ValidateConnectionAsync(Connection
         return await ValidateConnectionAsync(validateConnectionParams);
     }
 
+    public async Task<AdapterResponse<List<OrganizationDisplay>>> GetOrganizationsAsync(ICredentialsModel credentialsModel)
+    {
+        var failedResponse = new AdapterResponse<List<OrganizationDisplay>>(false, []);
+
+        return await threadHandling.RunOnBackgroundThread(async () =>
```

Comment:
I think it is better to be explicit.

Reply:
Eh, not really. You have the async lambda inside and you could add the Async suffix to the method name. Not a big deal in the end, but it would technically be more efficient to return just the task and not await it's result here

Reply:
I removed the async